internal/controller/certificates/certificates.go

```diff
@@ -18,9 +18,9 @@ package certificates
 
 import (
 	"context"
-	"slices"
 	"sort"
 
+	"golang.org/x/exp/slices"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
suggestion: this looks like an unrelated change? (If you're using an older version of go locally, you can run `make vendor-go` to use a vendored version which should enable you to compile and test locally)